docs/proposals/configmap.md

```diff
@@ -293,12 +296,12 @@ spec:
         - name: config-map-volume
           mountPath: /mnt/config-map
   volumes:
-  - name: config-map-volume
-    configMap:
-      files:
-        - path: "etc/redis.conf"
-          configMapName: redis-volume-config
-          key: redis.conf
+    - name: config-map-volume
+      configMap:
+        configMapName: redis-volume-config
```

Comment:
@thockin For LocalObjectReference we would need to default the value of the old field for old clients.  I can do that for both this and secrets in a follow-up if we want to do that now.

cc @smarterclayton @deads2k